metricbeat/module/logstash/node/_meta/data.json

```diff
@@ -4,23 +4,32 @@
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
+    "event": {
+        "dataset": "logstash.node",
+        "duration": 115000,
+        "module": "logstash"
+    },
+    "host": {
+        "hostname": "Shaunaks-MBP-2"
+    },
     "logstash": {
         "node": {
-            "host": "Shaunaks-MBP-2",
             "jvm": {
-                "pid": 3674,
-                "version": "1.8.0_171"
-            },
-            "version": "7.0.0-alpha1"
+                "pid": 39481,
```

Comment:
@ruflin @webmat Notice that I'm not deleting the original PID field here but instead just copying it over to the global `process.pid` field, per https://github.com/elastic/ecs#-process-fields:

> The `process.pid` often stays in the metric itself and is copied to the global field for correlation.

Can you verify that this looks good?

Reply:
I'd say it's fine to move it entirely. We don't yet have anything to represent runtime info such as VM/interpreter, language name and version; so the jvm stuff can stay there. But since there's only one process at play here, I don't think we need this other copy of the PID here.